code/game/jobs/jobs.dm

```diff
@@ -46,8 +46,7 @@ var/const/CARGOTECH			=(1<<7)
 var/const/MINER				=(1<<8)
 var/const/LAWYER			=(1<<9)
 var/const/CHAPLAIN			=(1<<10)
-var/const/CLOWN				=(1<<11)
-var/const/MIME				=(1<<12)
+var/const/VISITOR			=(1<<11)
 var/const/MERCHANT			=(1<<13)
```

Comment:
This should be: `var/const/MERCHANT			=(1<<12)`
The other entries should also have their numbers adjusted.

Reply:
I have been advised by Lohikar that changing those defines will fuck up the job preferences of everyone who is using them, and that they should never be changed once they have been set.